app/src/main/java/org/gem/indo/dooit/views/custom/NonSwipableViewPager.java

```diff
@@ -0,0 +1,57 @@
+package org.gem.indo.dooit.views.custom;
```

Comment:
The scroller seems to fix the animation on API 19, but not 17. I'm not aware of any change between the two versions that would cause this to happen.